packages/astro/src/@types/astro.ts

```diff
@@ -64,7 +64,7 @@ export type {
 	MarkdownHeading,
 	RehypePlugins,
 	RemarkPlugins,
-	ShikiConfig,
+	,
```

Comment:
It seems to me that the deletion is not desired:
```suggestion
	ShikiConfig,
```
(and I guess the build is failing because of that lonely comma)

Other than that, LGTM! There is an indentation issue but I guess the Format action will fix that, right?

Reply:
Perfect, thank you! And, I don't trust that indentation issues will be fixed? I guess it depends where it is!

Reply:
Must have been errant keystroke while I was searching for this term on the page. Did not mean to blow away shikiconfig!  :sweat_smile: